### ISSUE TITLE
Cannot Create Accessibility Report

### ISSUE THREAD

**Issue by brothman01** [joedolson/access-monitor#2]
I entered the Tenon API key and when I do that, instead of a report appearing, errors appear that are preventing the report from showing data.  The errors are:

Create Accessibility Report

Notice: Undefined property: stdClass::$info in /Users/ben/valet/test-site/wp-content/plugins/access-monitor/access-monitor.php on line 138

Notice: Undefined variable: errors in /Users/ben/valet/test-site/wp-content/plugins/access-monitor/access-monitor.php on line 143

Notice: Undefined property: stdClass::$info in /Users/ben/valet/test-site/wp-content/plugins/access-monitor/access-monitor.php on line 138

Notice: Undefined variable: errors in /Users/ben/valet/test-site/wp-content/plugins/access-monitor/access-monitor.php on line 143

Notice: Undefined property: stdClass::$info in /Users/ben/valet/test-site/wp-content/plugins/access-monitor/access-monitor.php on line 138

Notice: Undefined variable: errors in /Users/ben/valet/test-site/wp-content/plugins/access-monitor/access-monitor.php on line 143
Twenty Sixteen 1.3; Results from 3 pages tested

No errors found on http://test-site.dev.

No errors found on http://test-site.dev/my-account/.

No errors found on http://test-site.dev/2017/01/07/hello-world/.

**Comment by joedolson** [comment]
You can't run tests on a local site unless you create a connection to your
localhost environment using a tool like ngrok; see this article from Tenon:
http://blog.tenon.io/so-you-want-to-test-localhost-a-tutorial/

So while the notices are worth mentioning, and I'll look at them, they are
*not* what is preventing you from getting reports, unless you've already
set up the tunnel into your localhost environment.
